BigQuery/13_iv_antibiotics_pharmacy.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE TABLE `physionet.iv_abx_pharmacy` as
+
+select 
```

Comment:
Nit: Consistency on capitalization redux. Doesn't have to be done now, but keep it in mind.